secrets/test_util.go

```diff
@@ -0,0 +1,57 @@
+package secrets
+
+import (
+	"io/ioutil"
+	"net/url"
+	"os"
+	"testing"
+	"time"
+
+	clientv3 "go.etcd.io/etcd/client/v3"
+	"go.etcd.io/etcd/server/v3/embed"
+)
+
+// EtcdSetup is a helper that instantiates a new etcd cluster along with a
+// client connection to it. A cleanup closure is also returned to free any
+// allocated resources required by etcd.
+func EtcdSetup(t *testing.T) (*clientv3.Client, func()) {
```

Comment:
can remove the `etcdSetup` func that is still in the `secrets_test.go` file since it is no longer used